### PR TITLE
Plot up to 4 imu vibration metrics. Update color thresholds

### PR DESCRIPTION
**Pull request by AlexKlimaj**
Show up to 4 IMUs. Usually 2-3 internal and possibly 1 external IMU.

Change the color thresholds to be gravity and 1/2 gravity.

image

image

image

### CODE REVIEW

**Review by dagar — APPROVED** [review]

**Comment by dagar** [comment]
There are pylint failures, but otherwise looks good to me.
Screenshot 2023-01-15 at 3 42 00 PM

**Comment by AlexKlimaj** [comment]
@dagar fixed and pushed

File: app/plot_app/configured_plots.py [hunk 1]
```diff
@@ -640,10 +640,33 @@ def generate_plots(ulog, px4_ulog, db_data, vehicle_data, link_to_3d_page,
...
+                         x_range=x_range, y_start=0, topic_instance=0)
+    data_plot.add_graph(['accel_vibration_metric'], colors8[0:1],
+                         ['Accel 0 Vibration Level [m/s^2]'])
+
+    try:
+        data_plot.change_dataset('vehicle_imu_status', 1)
+        data_plot.add_graph(['accel_vibration_metric'], colors8[1:2],
+                             ['Accel 1 Vibration Level [m/s^2]'])
+    except:
+        pass
```
Comment (bkueng) [inline app/plot_app/configured_plots.py:652]: This shouldn't be needed

**Comment by bkueng** [comment]
@AlexKlimaj can you provide the links to the logs for the above examples?

**Comment by AlexKlimaj** [comment]
@bkueng 
https://review.px4.io/plot_app?log=8a2a2011-8b1c-47e3-bf9c-193da6f3b190
https://logs.px4.io/plot_app?log=aba3ca67-7721-4cc4-ac94-1b3a4c9f639e

**Pull request merged**